deps.edn

```diff
@@ -3,9 +3,9 @@
              "clojars" {:url "https://clojars.org/repo"}}
  :deps
  {org.clojure/clojure {:mvn/version "1.11.1"}
-  thheller/shadow-cljs {:mvn/version "2.19.8"}
+  thheller/shadow-cljs {:mvn/version "2.19.3"}
```

Comment:
This looks like a downgrade? Any particular reason to not use the newer version?